bin/universe.sh

```diff
@@ -24,6 +25,13 @@ configure_env() {
         echo "Using Spark dist URI: $SPARK_DIST_URI"
     fi
 
+    if [ -z "${CLI_VERSION}" ]; then
```

Comment:
Let's remove the option to customize it via an env var.  That shouldn't be needed.